src/cockpit/samples.py

```diff
@@ -220,11 +219,11 @@ def sample(self, samples: Samples) -> None:
                     _,
                     dev_name,
                     _,
-                    num_reads_merged,
+                    _num_reads_merged,
```

Comment:
This should then be consistent and use `_` everywhere. Moreover, it then seems nicer to directly pick the 4 values that we care about from the array positions, instead of having this soup of `_` and many wasted lines.